### PR TITLE
Add make install and make uninstall rules to cmake

### PR DESCRIPTION
**Pull request by PureTryOut**

### CODE REVIEW

**Comment by mujx** [comment]
Could you move the `cmake_uninstall` file in the `cmake` directory?

File: CMakeLists.txt [hunk 1]
```diff
@@ -337,3 +339,24 @@ else()
     add_executable (nheko ${OS_BUNDLE} ${NHEKO_DEPS})
     target_link_libraries (nheko ${NHEKO_LIBS} Qt5::Multimedia)
 endif()
+
+# Install target
```
Comment (mujx) [inline CMakeLists.txt]: Could you wrap this whole section in a `if (LINUX) ... endif(LINUX)` block to fix the build on macos? Any other fix is also welcome.

**Comment by PureTryOut** [comment]
I used a `if(UNIX AND NOT APPLE)` block instead of `if(LINUX)` as with the latter, the install rule was not added to the makefile.

**Comment by mujx** [comment]
Dank je!

**Pull request merged**